q2_fmt/_peds.py

```diff
@@ -739,65 +741,109 @@ def _create_sim_masking(mismatched_df, donor_df, reference_column):
     return donor_mask
 
 
-def _simulate_uniform_distro(mismatched_peds, iterations):
+def _simulate_uniform_distro(mismatched_peds, k):
     """Randomly samples the mismatched PEDS values.
 
     Creates a uniform distribution of mismatched PEDS values by randomly
-    sampling `iterations` times with replacement.
+    sampling `num_iterations` times with replacement.
 
     Parameters
     ----------
     mismatched_peds: pd.DataFrame
         A Dataframe that contains all mismatched PEDS values.
-    iterations: int
-        Number of iterations to run simulations (Number of times to randomly
-        sample mismatched_peds)
+    k: int
+        Number of iterations(`k`) to run simulations (Number of times to
+        randomly sample mismatched_peds)
 
     Returns
     -------
     peds_iters: pd.Series
-        a pd.Series with all iterations of mismatched PEDS Values. This will
-        later be compared to an actual PEDS value.
+        a pd.Series with all num_iterations of mismatched PEDS Values. This
+        will later be compared to an actual PEDS value.
 
     Examples
     --------
-    mismatched_peds = [0, 0, 0,0 ]
-    iterations = 10
+    >>> mismatched_peds = [0, 0, 0,0 ]
+    >>> num_iterations = 10
 
-    _simulate_uniform_distro(mismatched_peds, iterations)
+    >>> _simulate_uniform_distro(mismatched_peds, num_iterations)
 
     pd.Series([data = [0, 0, 0, 0, 0, 0, 0, 0, 0],
                index = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]])
     """
-    peds_iters = random.choices(mismatched_peds, k=iterations)
+    peds_iters = random.choices(mismatched_peds, k=k)
     # Tranforming to series for easy series math in _per_subject_stats()
     peds_iters = pd.Series(peds_iters)
     return peds_iters
 
 
-def _per_subject_stats(mismatched_peds, actual_temp,
-                       iterations, mismatched_pairs_n):
+def peds_sim_stats(value, peds_iters, num_iterations):
```

Comment:
Instead of passing in `num_iterations`, compute that internally here as `len(peds_iters)` to avoid a value being provided for `num_iterations` that doesn't match the number of values in `ped_iters`.